cloud-src/modules/ROOT/pages/quickstart.adoc

```diff
@@ -0,0 +1,100 @@
+= Quickstart guide
+:keywords: start, typedb, cloud, quickstart, tutorial, registration
+:longTailKeywords: getting started, get started with typedb cloud, typedb cloud tutorial, typedb cloud quickstart
+:pageTitle: Quickstart guide
+:summary: How to start using TypeDB Cloud.
+:experimental:
+
+== Overview
+
+In this Quickstart guide, we will find out how to start using TypeDB Cloud:
+
+* Get an account
+* Log In
+* Create a deployment
+* Connect to a TypeDB Enterprise cluster
+
+== Get an account
+
+include::cloud::account.adoc[tag=create-account]
+
+== Log in
+
+include::cloud::account.adoc[tag=log-in]
+
+== Create a deployment
+
+include::cloud::deployments/management.adoc[tag=create]
+
+Now that the deployment is being created all we need is to wait for the status of the deployment to become
+`Fully available`.
+
+== Connect to a deployment
+
+To work with the deployment's TypeDB servers, we need to set up a TypeDB Client and connect it to the
+deployment cluster.
+
+// tag::default-credentials[]
+[NOTE]
+====
+At this stage of alpha testing, all TypeDB Cloud deployments are created with the listening port of `1729`
+and a default user with the following credentials:
+
+Username: `admin` +
+Password: `password`
+====
+// end::default-credentials[]
+
+To connect to a TypeDB Cloud deployment we can use any TypeDB Client: see the xref:deployments/connect.adoc[] page
+for more options.
+
+For example, to connect via TypeDB Console:
+
+// tag::console-connect[]
+[,bash]
+----
+typedb console --cluster=<address> --username=admin --password=password --tls-enabled=false
+----
+
+where:
+
+*  `<address>` -- is the address of a server in the TypeDB Cloud installation we are connecting to,
+including the port number. +
+For example, `a242676f5710e462e9eed7a78d795ecb-196767772.eu-west-2.elb.amazonaws.com:1729`.
+* `admin` -- is the default username for the default user.
+* `password` -- is the default password for the default user.
+* `false` -- is the TLS encryption setting. It's false (disabled) by default for now, but that can change soon as
+TypeDB Cloud is in alpha testing stage.
```

Comment:
By the time we release this documentation, it should be the case that deployments are only encrypted.

Reply:
Fixed in https://github.com/vaticle/typedb-docs/pull/686/commits/9dd696e44c70151070a284dc0e90b7708c493fb7